package com.clp.CLPValidation.Task.exception;

public class UserNotFoundException extends Exception {

	public UserNotFoundException(String message) {
		super(message);
	}
}
